CSharp/BatchExplorer/Helpers/BasicDataProvider.cs

```diff
@@ -1,137 +1,151 @@
-﻿//Copyright (c) Microsoft Corporation
```

Comment:
This commit (hash 29019bae098fd83b1955425e49968b30f13fa6d4) seems to have a lot of changes (most of which look like they are due to line-ending fixups).

Can you back-out 29019bae098fd83b1955425e49968b30f13fa6d4 from your PR and just keep the other 3 commits for now, which we can merge quickly, and then we can discuss this larger change as a separate PR?